proposals/nnnn-closure-isolation-control.md

```diff
@@ -80,7 +80,7 @@ Enable explicit specification of actor-isolation via an isolated parameter in a
 
 ```swift
 actor A {
-  func isolate() {
+  nonisolated func isolate() {
```

Comment:
I was a little confused here, because I think the same is semantically equivalent to just not using an isolated capture at all, isn't it?

I make this function `nonisolated` to really highlight the new capabilities. But please look at this closely, because I could be wrong!